Wikipedia/Wikipedia-Info.plist

```diff
@@ -65,6 +67,7 @@
 	<key>UIBackgroundModes</key>
 	<array>
 		<string>fetch</string>
+		<string>processing</string>
```

Comment:
@joewalsh Just curious as to your reasons for adding this - I see an error on the build server log with ""Missing Info.plist value. The Info.plist key 'BGTaskSchedulerPermittedIdentifiers' must contain a list of identifiers used to submit and handle tasks when 'UIBackgroundModes' has a value of 'processing'. For more information, refer to the Information Property List Key Reference at https://developer.apple.com/library/ios/documentation/General/Reference/InfoPlistKeyReference/Introduction/Introduction.html.""

Reply:
There was an issue with `WKWebView` without it but that was a few betas ago. I'll remove it and see if the error persists. If it does, there's a `BackgroundTasks` branch that converts our background fetch to a background task and adds the appropriate identifier that we could incorporate.